src/sa_pool.c

```diff
@@ -292,9 +331,51 @@ static int sa_pool_free_hash(struct sa_pool *ap)
     return EDPVS_OK;
 }
 
-int sa_pool_create(struct inet_ifaddr *ifa, uint16_t low, uint16_t high)
+static int __sa_pool_create(struct inet_ifaddr *ifa, lcoreid_t cid,
+                            uint16_t low, uint16_t high)
 {
+    uint32_t filtids[MAX_FDIR_PROTO];
+    struct sa_fdir *fdir = &sa_fdirs[cid];
     struct sa_pool *ap;
+    int err;
+    ap = rte_zmalloc(NULL, sizeof(struct sa_pool), 0);
+    if (!ap) {
+        err = EDPVS_NOMEM;
+        goto errout;
+    }
+
+    ap->ifa = ifa;
+    ap->low = low;
+    ap->high = high;
+    rte_atomic32_set(&ap->refcnt, 0);
+
+    err = sa_pool_alloc_hash(ap, sa_pool_hash_size, fdir);
```

Comment:
Hi,
Do you still need to change this function according to pool_mode? As the some port is masked in lport_lcore_mapping_mode.